ext/nnstreamer/tensor_decoder/tensordec-octetstream.c

```diff
@@ -0,0 +1,134 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * GStreamer/NNStreamer Tensor-Decoder
+ * Copyright (C) 2021 Gichan Jang <gichan2.jang@samsung.com>
+ */
+/**
+ * @file	tensordec-octetstream.c
+ * @date	04 Nov 2021
+ * @brief	NNStreamer tensor-decoder subplugin, "octet strea",
+ *              which converts tensors to octet stream.
+ * @see		https://github.com/nnstreamer/nnstreamer
+ * @author	Gichan Jang <gichan2.jang@samsung.com>
+ * @bug		No known bugs except for NYI items
+ *
+ */
+
+#include <string.h>
+#include <glib.h>
+#include <nnstreamer_plugin_api_decoder.h>
+#include <nnstreamer_plugin_api.h>
+#include <nnstreamer_log.h>
+#include <nnstreamer_util.h>
+#include "tensordecutil.h"
+
+G_BEGIN_DECLS
```

Comment:
I'm not sure `G_BEGIN_DECLS` (which is `extern "c"`) is meaningful in a source file.

Reply:
Right. It's unnecessary. Removed. Thanks.